src/deploy.ts

```diff
@@ -22,70 +22,102 @@ async function run() {
   const STACK_TYPE = process.env.STACK_TYPE || 'do-k8s-cdktf';
   const STACK_TEAM = process.env.OPS_TEAM_NAME || 'private'
 
+  const { SRV_TYPE } = await ux.prompt<{
+    SRV_TYPE: string
+  }>({
+    type: 'list',
+    name: 'SRV_TYPE',
+    choices: ['app', 'util'],
+    default: 'app',
+    message: 'Which type of application?'
+  })
+
+  let UTIL_NAME: string = '';
+  let STACK_TAG: string = '';
+  let STACK_REPO: string = '';
+  if (SRV_TYPE === "util") {
+    ({ UTIL_NAME } = await ux.prompt<{
+      UTIL_NAME: string
+    }>({
+      type: 'list',
+      name: 'UTIL_NAME',
+      choices: ['dora-controller'],
```

Comment:
this value 'dora-controller' could be a constant, it's used several times into the files src/destroy.ts, src/deploy.ts, and src/stack/dora.ts

Reply:
Done